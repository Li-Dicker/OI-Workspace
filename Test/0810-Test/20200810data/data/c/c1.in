50
0 1 2 1 2 1 3 5 7 1 1 2 0 5 14 15 15 6 9 18 10 13 12 10 9 1 17 10 1 2 6 14 23 6 18 19 13 29 35 8 13 20 0 25 41 11 41 24 9 31
11 5 11 42 23 9 5 11 11 48 11 42 42 16 11 5 9 42 9 11 11 42 11 42 10 16 11 16 10 16 11 16 5 16 16 11 11 11 11 5 16 5 16 11 11 5 16 11 5 5
