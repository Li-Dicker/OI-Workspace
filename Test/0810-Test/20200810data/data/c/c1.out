0 0 1 13 0 0 2 1 3 0 1 13 0 43 1 2 6 13 6 37 1 13 37 13 29 43 31 43 0 43 1 43 2 43 41 31 1 1 45 2 43 2 0 38 1 2 43 37 7 2 
